TestSuite/pack.py

```diff
@@ -650,3 +656,8 @@ def create_trigger(self, name, content: dict = {}) -> Trigger:
         trigger = Trigger(name, self._triggers_path, content)
         self.triggers.append(trigger)
         return trigger
+
+    def create_agent_config(self, name, json_content: dict = {}, yaml_content: dict = {}) -> AgentConfig:
```

Comment:
https://florimond.dev/blog/articles/2018/08/python-mutable-defaults-are-the-source-of-all-evil/
```suggestion
    def create_agent_config(self, name, json_content: dict = None, yaml_content: dict = None) -> AgentConfig:
```